Drift/Private/DriftFlexmatch.cpp

```diff
@@ -50,41 +56,72 @@ TStatId FDriftFlexmatch::GetStatId() const
 
 void FDriftFlexmatch::ReportLatencies()
```

Comment:
This method could be re-entered if the collective task of pinging and reporting takes longer than the ping interval, is that, intentional, or should the next ping always happen *after*?

Reply:
Semi-intentional; was aware of if but didn't see any immediate issues with leaving it as is, so I decided not to deal with it now...

Reply:
fixed it